qiling/os/windows/dlls/kernel32/tlhelp32.py

```diff
@@ -28,7 +28,7 @@ def hook_CreateToolhelp32Snapshot(ql, address, params):
     # TODO thinking about implementing an handler, gonna see if is really necessary
     flag = params["dwFlags"]
     if flag == TH32CS_SNAPPROCESS:
-        ql.dprint(0,"[.] We probably found a Ransomware!")
+       ql.dprint(0, "[!] We probably found a Ransomware!")
```

Comment:
this is maybe a "decision-able"  statement, so a brand new symbol [.] or something can be fine. why a new symbol, i do mention in another comment

Reply:
f4cf13f should resolve the problem